extras/ci/gh_api.py

```
@@ -8,12 +8,46 @@
+        "gateway timeout",
+        "secondary rate limit",
+        "api rate limit exceeded",
+    ]
+    return any(marker in text for marker in retry_markers)
+
+
+def _run_gh_command(cmd):
+    """Run a gh command with retry for transient failures."""
+    last_result = None
+    for attempt in range(MAX_RETRIES):
+        result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)